remoting/etcdv3/client_test.go

```diff
@@ -0,0 +1,365 @@
+package etcdv3
+
+import (
+	"fmt"
+	"go.etcd.io/etcd/embed"
+	"net/url"
+	"path"
+	"reflect"
+	"strings"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/suite"
+
+	"github.com/coreos/etcd/mvcc/mvccpb"
+	"github.com/juju/errors"
+	"google.golang.org/grpc/connectivity"
```

Comment:
format  import and don't use juju/errors。

Reply:
yeah.

Reply:
Thx, i had reformat all import && change juju/errors to  pkg/errors.